python/src/aac/plugins/plugin_manager.py

```diff
@@ -67,6 +67,7 @@ def get_plugin_manager() -> PluginManager:
         "subcomponent_type",
         "unique_names",
         "unique_requirement_ids",
+        "unused_definitions",
```

Comment:
The way these get registered have changed from some of the previous validator creations, and I was only seeing this list that I needed to update within the manager.

Reply:
Yep! Obviously this won't matter, but since we're getting the info messages in other tests it's demonstrating that the validator is correctly registered. 👍